postgresql-11.5/src/bin/psql/commandbteq.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * psql - the PostgreSQL interactive terminal
+ *
+ * Copyright (c) 2000-2018, PostgreSQL Global Development Group
+ *
+ * src/bin/psql/command.h
+ */
+#ifndef COMMAND_H
+#define COMMAND_H
+
+#include "fe_utils/print.h"
+#include "fe_utils/psqlscan.h"
+#include "fe_utils/conditional.h"
+
+
+typedef enum _dotResult
+{
+	PSQL_CMD_UNKNOWN = 0,		/* not done parsing yet (internal only) */
+	PSQL_CMD_SEND,				/* query complete; send off */
+	PSQL_CMD_SKIP_LINE,			/* keep building query */
+	PSQL_CMD_TERMINATE,			/* quit program */
+	PSQL_CMD_NEWEDIT,			/* query buffer was changed (e.g., via \e) */
+	PSQL_CMD_ERROR				/* the execution of the dot command
+								 * resulted in an error */
```

Comment:
Could you, please, update enum members to be as `BTEQ_CMD_*` as they may conflict with each other.

Reply:
OK